content/blog/deploying-cribl-stream-containers-on-hpe-greenlake.md

```diff
@@ -0,0 +1,224 @@
+* **Management simplicity:** HPE GreenLake provides a single management console for managing your Cribl Stream containers, making it easy to keep your observability pipeline running smoothly.
+
+![Cribl architecture diagram](/img/criblarchitecure.png "Cribl architecture")
+
+#### Prerequisites
+
+Before you deploy Cribl Stream containers on HPE GreenLake, you will need to:
+
+* Have a HPE GreenLake account on <https://common.cloud.hpe.com/>.
+* Install the HPE GreenLake CLI.
+* Create a HPE GreenLake cluster.
+* Install the Cribl Stream Kubernetes operator.
+
+Steps to deploy Cribl Stream containers on HPE GreenLake:
+
+1. Create a Cribl Stream deployment file. This file will specify the Cribl Stream containers that you want to deploy, as well as the resources that they need.
+2. Deploy the Cribl Stream containers to your HPE GreenLake cluster using the Cribl Stream Kubernetes operator.
+3. Verify that the Cribl Stream containers are running and healthy.
+4. Configure Cribl Stream to collect and process your data.
+5. Send your data to your analysis platform of choice.
+
+#### Example deployment file
+
+The following example deployment file deploys a Cribl Stream container that collects and processes logs from a Kubernetes cluster:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cribl-stream
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cribl-stream
+  template:
+    metadata:
+      labels:
+        app: cribl-stream
+    spec:
+      containers:
+      - name: cribl-stream
+        image: cribl/cribl-stream:latest
+        ports:
+        - containerPort: 9000
+        volumeMounts:
+        - name: cribl-stream-config
+          mountPath: /etc/cribl-stream
+      volumes:
+      - name: cribl-stream-config
+        configMap:
+          name: cribl-stream-config
+```
+
+#### Deploying Cribl Stream using Helm Charts
+
+The Cribl Stream helm charts can be found on github (<https://github.com/criblio/helm-charts>). This assumes that the namespace is set to `cribl`. 
+
+﻿Log into cloud CLI or jump box and issue the following commands:
+
+```shell
+export KUBECONFIG=<path_to_kube_settings>
+kubectl get nodes -n cribl
+kubectl get svc  -n cribl
+```
+
+Label the leader node and the worker nodes:
+
+```shell
+kubectl label nodes <leader_node> stream=leader
+kubectl label nodes <worker_node> stream=worker
+```
+
+Validate by running:
+
+```shell
+kubectl get nodes --show-labels
+```
+
+Create and modify the `values.yaml` file for workers and leader nodes. For the leader nodes, create a file named `Leader_values.yaml` and modify line 97:
+
+```yaml
+nodeSelector:
+     stream: leader
+```
+
+For the worker nodes, create a file named `Worker_values.yaml` and modify line 97:
+
+```yaml
+nodeSelector:
+     stream: worker
+```
+
+Next set the labels for your workers and leader node.
+
+First get a list of all the nodes and see the labels associated with them:
```

Comment:
Consider rephrasing this to:
"To do this, you'll first need to get a list of all the nodes and the labels associated with them." 

(The reason I'm suggesting this is because of the two statements, one right after the other that read:
"Next, ...
"First, ...

Stylistically, it's kind of awkward.